src/bindgen/typescript-resources/template.package.json

```diff
@@ -42,6 +42,7 @@
     "rollup-plugin-copy": "^3.4.0",
     "rollup-plugin-ignore": "^1.0.10",
     "rollup-plugin-polyfill-node": "^0.11.0",
+    "tslib": "^2.5.0",
```

Comment:
tslib, cool.  But not using yet right?

Reply:
I had to add it to avoid build errors with MeshToPolyData :shrug: . If there is a way to get rid of it, let's do it.